protocol/protocol.md

```diff
@@ -270,33 +305,20 @@ Reponse: 207
 
 Response body example:
 
-    <?xml version='1.0' encoding='utf-8'?>
+    <?xml version='1.0' encoding='UTF-8'?>
     <d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
-      <d:response>
-        <d:href>/oc/remote.php/webdav/</d:href>
-        <d:propstat>
-          <d:prop>
-            <oc:id>00000003ocobzus5kn6s</oc:id>
-            <oc:permissions>RDNVCK</oc:permissions>
-            <oc:size>592610518</oc:size>
-            <d:getetag>"55ed918c28ac9"</d:getetag>
-          </d:prop>
-          <d:status>HTTP/1.1 200 OK</d:status>
-        </d:propstat>
-      </d:response>
-       <d:response>
-        <d:href>/oc/remote.php/webdav/%d7%91%d7%a2%d7%91%d7%a8%d7%99%d7%aa-.txt</d:href>
+    <d:response>
+        <d:href>/oc9/remote.php/webdav/</d:href>
```

Comment:
Is 'oc9' component on purpose here?

I think 'oc' component was a mistake in the first place (we should probably include the request URL in the example and use it in the example response, something like: <d:href>example.org/remote.php/webdav/... )